internal/pkg/skuba/addons/dex.go

```diff
@@ -45,14 +46,7 @@ func renderDexTemplate(addonConfiguration AddonConfiguration) string {
 
 type dexCallbacks struct{}
 
-func (dexCallbacks) beforeApply(addonConfiguration AddonConfiguration, skubaConfiguration *skuba.SkubaConfiguration) error {
-	var err error
```

Comment:
Do you no longer need `var err` here, since `err` is still used  in the cert exists check?

Reply:
I guess we don't declare variables anywhere else, so...  Nevermind. :)